### ISSUE TITLE
Fresh install of asdf and asdf-ruby, 'list-all' shows git info, no versions

### ISSUE THREAD

**Issue by colinta** [asdf-vm/asdf-ruby#142]
When I run `asdf list-all ruby` I get a bunch of git output, and no versions.  I tried uninstalling and reinstalling the plugin (included in this output)

```
> asdf plugin-remove ruby
> asdf plugin-add ruby
> asdf list-all ruby
Downloading ruby-build...
Cloning into 'ruby-build-source'...
remote: Enumerating objects: 1, done.
remote: Counting objects: 100% (1/1), done.
remote: Total 9941 (delta 0), reused 0 (delta 0), pack-reused 9940
Receiving objects: 100% (9941/9941), 2.11 MiB | 11.53 MiB/s, done.
Resolving deltas: 100% (6449/6449), done.
Note: checking out 'v20191004'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at da8e0b4 Bump version to v20191004
> 
```

**Comment by Stratus3D** [comment]
Part of this has been addressed by #145.

**Comment by colinta** [comment]
I still get some git output but maybe that's intended - the ruby versions do list out after the git output.  Closing, and I'll include some of the output here for reference / google-search ability:

```
asdf list all ruby
Downloading ruby-build...
Cloning into '/Users/colinta/.asdf/plugins/ruby/ruby-build-source'...
remote: Enumerating objects: 360, done.
remote: Counting objects: 100% (360/360), done.
remote: Compressing objects: 100% (256/256), done.
remote: Total 10619 (delta 257), reused 134 (delta 88), pack-reused 10259
Receiving objects: 100% (10619/10619), 2.23 MiB | 12.47 MiB/s, done.
Resolving deltas: 100% (6938/6938), done.
Note: checking out 'v20200115'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 901d0e9 ruby-build 20200115
1.8.5-p52
1.8.5-p113
1.8.5-p114
...
```
